GOVERNANCE.md

```diff
@@ -0,0 +1,33 @@
+# Governance
+
+## Project Maintainers
+
+[Project maintainers](CODEOWNERS) are responsible for all activities
+maintaining and updating the Notary project. Final decisions on the projects
+reside with the project maintainers.
+
+Maintainers MUST remain active. To remain active they MUST collaborate on the 
+project at least once in a 3 month period. Example collaborations include, attending
+the community meeting, creation or commenting on issues/PRs, PR reviews, etc. 
+If they are unresponsive for >3 months, they will be automatically moved to emeritus unless a
+[super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of
+the other project maintainers agrees to extend the period to be greater than 3
+months.
+
+New maintainers can be added to the project by a
+[super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote)
+vote of the existing maintainers. A potential maintainer may be nominated by an existing maintainer.
+Submissions for new maintainers can be made by creating an
+[issue](https://github.com/notaryproject/notaryproject/issues/new). The current maintainers will cast
+their vote via a +1 comment on the issue. Once a [super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote)
```

Comment:
> The current maintainers will cast their vote via a +1 comment on the issue.
How many votes can be cast? 

Also, If going off by the codeowners file being introduced as part of this PR, restricting voting to two individuals makes it that supermajority has to be the absolute majority.  Worth considering extending the voting to all community members and contributors with demonstrated active engagement.

Reply:
+1

Reply:
> How many votes can be cast?

A single vote per maintainer. (I will update)

> Also, If going off by the codeowners file being introduced as part of this PR, restricting voting to two individuals makes it that supermajority has to be the absolute majority. Worth considering extending the voting to all community members and contributors with demonstrated active engagement.

It might be worth using the linked issue as a seed set of maintainers before putting this into effect so that we can establish a super-majority as noted. We may have to add some verbiage about how the seed maintainers will be selected also. I'll add this to the agenda for the next meeting.

Reply:
It's also worth pointing out that the initial maintainers must show a track record of having been _publicly_ actively involved in the project so far. I'm afraid that at least one of them has not.

Reply:
Fair point - Do you of any other CNCF projects that have a maintainer definition outside of the spiffe project as I'm taking a look around to see if there is good precedence on maintainer requirements that this project might consider adopting. I also see https://github.com/openservicemesh/osm/blob/main/CONTRIBUTING_LADDER.md#maintainer

Reply:
The closest I can find so far is the [TUF governance model](https://github.com/theupdateframework/tuf/blob/develop/docs/GOVERNANCE.md). We have the equivalent of a BDFL who necessarily must remain active in order to remain one. Otherwise, maintainers and the community can initiate, vote for, and enact changes.